include/mxnet/lib_api.h

```diff
@@ -393,13 +458,21 @@ class OpResource {
     return static_cast<mx_stream_t>(cuda_stream);
   }
 
+  void alloc_ndarray(MXSparse* sparse, int index, int indices_len, int indptr_len = 0) {
```

Comment:
should we call this "alloc_sparse" so that users dont try and use it for general ndarray allocation? especially since it takes a sparse tensor as input.

Reply:
Make sense. Updated.